client/components/happychat/style.scss

```diff
@@ -52,6 +52,10 @@
 	right: 12px;
 	font-size: 14px;
 	display: none;		/* disabled */
+	box-sizing: border-box;
+
+	transform: translateZ(0);
```

Comment:
I'm asking because last I used `will-change`, there were some situations on mobile Safari where it didn't appear to work. But that's six or so months ago, so things might've changed.